development/carma.env

```diff
@@ -33,3 +33,6 @@ export CARMA_LOCZ_NS="/localization"
 
 # Namespace of nodes in the web ui stack
 export CARMA_UI_NS="/ui"
+
+# configuration of CycloneDDS
+export CYCLONEDDS_URI=/opt/carma/maps/cyclone_config.xml
```

Comment:
This is putting the xml file in /opt/carma/maps. I can see why the prototype might have done that, but it doesn't make a ton of sense long term. Instead put it directly in the config so that it wouldn't need a volume. 

Better to add a copy of the xml file for all the vehicles to the config and have that pointed to by the environment variable.

Reply:
additionally the change set should be applied to all vehicles